tool/tsh/mfa.go

```diff
@@ -663,3 +671,24 @@ func showOTPQRCode(k *otp.Key) (cleanup func(), retErr error) {
 		}
 	}, nil
 }
+
+func deleteTouchIDCredentialIfApplicable(credentialID, rpID string) {
+	exitst, err := touchid.CredentialExists(credentialID, rpID)
+	if err != nil {
+		if errors.Is(err, touchid.ErrCredentialNotFound) || errors.Is(err, touchid.ErrNotAvailable) {
+			// Nothing to do here, just proceed.
+			return
+		}
+		log.WithError(err).Errorf("Failed to check if touchID credential exists: %s\n", credentialID)
+		return
+	}
+	if !exitst {
+		log.Debugf("Credential %s does not exists, can't delete\n", credentialID)
+		return
+	}
+	fmt.Println("Deleting touchID credential")
+	if err := touchid.DeleteCredential(credentialID); err != nil {
```

Comment:
This would prompt the user without warning, which I think is a bit strange. I think a non-interactive delete is alright here.

Reply:
done